src/widgets.ts

```diff
@@ -83,6 +85,38 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with the iframe widget about a navigation event
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget
+   * The iframe widget can listen for the 'mx/navigation' event
+   * And send its own post message event back with a `did_go_back` property.
+   */
```

Comment:
Nit: can you just capitalize the first letter of each sentence rather than each line?

Reply:
Or add missing periods I guess.